lookout/style/format/benchmarks/generate_smoke.py

```diff
@@ -103,14 +104,20 @@ def generate_smoke_entry(inputpath: str, outputpath: str, force: bool = False) -
         writer = csv.DictWriter(index_file, fieldnames=["repo", "style", "from", "to"])
         writer.writeheader()
         for style_name, (pattern, repl) in js_format_rules.items():
-            # TODO(zurk): handle custom modification functions as well as regexp.
-            pattern = re.compile(pattern)
+            if isinstance(pattern, str):
```

Comment:
I am not sure about its signature (input parameters mostly). For now, I think that `repl ` argument is redundant and it should look like 
```
new_style_code = modify_style(initial_code)
```

I think we can keep this TODO in code as soon as we do not have any function for style modification.
When (and If) we actually need style modification functions, we can resolve it with a better understanding of what we actually need.

If we want to close all todos in the code we can just delete it. I think it is a minor one.